src/topology/algebra/uniform_group.lean

```diff
@@ -3,10 +3,12 @@ Copyright (c) 2018 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Johannes Hölzl
 -/
-import topology.uniform_space.uniform_embedding
-import topology.uniform_space.complete_separated
-import topology.algebra.group
+
 import tactic.abel
+import topology.algebra.group
+import topology.uniform_space.complete_separated
+import topology.uniform_space.uniform_convergence
+import topology.uniform_space.uniform_embedding
```

Comment:
Is the reordering and additional blank line intentional? How about just adding `import topology.uniform_space.uniform_convergence`?